cmd/bleve/cmd/zap/docvalue.go

```diff
@@ -255,7 +253,19 @@ func getDocValueLocs(docNum uint64, metaHeader []zap.MetaData) (uint64, uint64)
 		return metaHeader[i].DocNum >= docNum
 	})
 	if i < len(metaHeader) && metaHeader[i].DocNum == docNum {
-		return metaHeader[i].DocDvLoc, metaHeader[i].DocDvLen
+		var start, end uint64
+		if i > 0 {
+			start = metaHeader[i].DocDvOffset
+		}
+		// single element case
```

Comment:
not that this is incorrect, but it feels like this should be doable or simplify'able into two cases instead of three?  that is, we're at the end, or we're not at the end?

Reply:
not sure @steveyen , can you please check the loadChunk PR: #815 . It tries to explain the single offset slice logic.

Reply:
i see - thx... will leave commentary on the other PR #815